### DISCUSSION TITLE
Deleting the old repo, and reconnecting the new repo had issues

### DISCUSSION THREAD

**Discussion by hiifong** [vercel/community#1215] [Help] [answered]
### Summary

Error:The provided GitHub repository does not contain the requested branch or commit reference. Please ensure the repository is not empty.

**Comment by amyegan** [comment]
Thanks for reaching out @hiifong. If you are not the owner of the new repo, please make sure you have "collaborator" access to avoid a [missing git repository](https://vercel.com/docs/concepts/git/vercel-for-github#missing-git-repository) error.

You should also double check the Application permissions for Vercel on your GitHub account. If you have chosen "Only select repositories" in the Repository Access section, then you need to make sure the new repo appears in the selected list. Then Vercel will be able to access the new repo for future deployments. You can find this in your GitHub Settings > Applications > Vercel.

<img src="https://user-images.githubusercontent.com/8247105/211040959-79234378-7c7a-4560-8ece-e2b126e2a099.png" alt="Screenshot of repository access settings showing three selected repositories" />
 
If you tried this and it still isn't working, please share more info about the steps you took so I can try to reproduce the error 🙏

**Reply by hiifong** [reply]
Thank you！
I used the [vercel template](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fwalinejs%2Fwaline%2Ftree%2Fmain%2Fexample) before, and I deleted the git repository corresponding to this template just now, and then I forked this template again, the repository name is the same as before, and then re-associated this repository in the project, and this problem occurred.

**Comment by hiifong** [comment, marked as answer]
Thank you！
I have solved this problem, when I update the git repo, vercel will actively clone the repo again, and the problem is solved
